packages/teams-js/src/public/monetization.ts

```diff
@@ -9,27 +10,41 @@ import { runtime } from './runtime';
  */
 export namespace monetization {
   /**
-   * @hidden
+   * @privateRemarks
```

Comment:
Please revert to @hidden -- it's the preferred way that actually works to hide this from the public docs. Turns out that @privateRemarks did not work with the MS Docs pipeline.

Reply:
Done